bokehjs/src/lib/api/palettes.ts

```diff
@@ -643,6 +643,183 @@ export const Colorblind6 = [0x0072b2ff, 0xe69f00ff, 0xf0e442ff, 0x009e73ff, 0x56
 export const Colorblind7 = [0x0072b2ff, 0xe69f00ff, 0xf0e442ff, 0x009e73ff, 0x56b4e9ff, 0xd55e00ff, 0xcc79a7ff]
 export const Colorblind8 = [0x0072b2ff, 0xe69f00ff, 0xf0e442ff, 0x009e73ff, 0x56b4e9ff, 0xd55e00ff, 0xcc79a7ff, 0x000000ff]
 
+export const Bright3 = [0x4477aaff, 0xee6677ff, 0x228833ff]
+export const Bright4 = [0x4477aaff, 0xee6677ff, 0x228833ff, 0xccbb44ff]
+export const Bright5 = [0x4477aaff, 0xee6677ff, 0x228833ff, 0xccbb44ff, 0x66cceeff]
+export const Bright6 = [0x4477aaff, 0xee6677ff, 0x228833ff, 0xccbb44ff, 0x66cceeff, 0xaa3377ff]
+export const Bright7 = [0x4477aaff, 0xee6677ff, 0x228833ff, 0xccbb44ff, 0x66cceeff, 0xaa3377ff, 0xbbbbbbff]
```

Comment:
In cases where the next palette only appends a value, one can use the following to reduce duplication and JS size:
```suggestion
export const Bright7 = [...Bright6, 0xbbbbbbff]
```

Reply:
Thank you for the suggestion. Should I apply this to all the other palettes? I saw some previously defined colours were also written with duplications.

Reply:
It would definitely be a good thing to apply this pattern everywhere that it can be applied